### PR TITLE
Wording edits to staking docs

### PR DESCRIPTION
**Pull request by joshuahannan**
## Description

Makes edits to wording and spelling of staking docs

### CODE REVIEW

**Review by 10thfloor — APPROVED** [review]

File: docs/content/staking/index.mdx [hunk 1]
```diff
@@ -31,18 +31,18 @@ The list contains important information about each node, like their public keys,
 and what kind of node they are running. 
 (Collection, Consensus, Execution, Verification, or Access)
 
-To register their node, the node operator submits a transaction to register their node along with tokens they want to stake.
+To register their node, the node operator submits a transaction to register their node along with the tokens they want to stake.
```
Comment (psiemens) [inline docs/content/staking/index.mdx:34]: "register their node" is repeated twice, can just say 

> A node operator registers a node by submitting a transaction containing their node information and the FLOW they wish to stake.

File: docs/content/staking/index.mdx [hunk 2]
```diff
@@ -31,18 +31,18 @@ The list contains important information about each node, like their public keys,
...
 
-Once a node operator is registered and staked, assuming they keep their node online, are well behaved, 
-and are consistent maintainers of the network, they receive rewards at the end of every week. 
+Once a node operator is registered and staked-assuming they keep their node online, are well behaved, 
+and are consistent maintainers of the network-they receive rewards at the end of every week.
```
Comment (psiemens) [inline docs/content/staking/index.mdx:37]: this sentence runs on a bit, I'd consider trimming it down a bit

> A staked node will receive weekly reward payments at the end of each epoch, assuming it stays online and actively participates in the protocol without committing any slashable offences.

File: docs/content/staking/index.mdx [hunk 3]
```diff
@@ -73,7 +73,7 @@ and the amount of tokens they want to delegate.
 The tokens are committed and managed in the exact same way that normal staked tokens are managed. 
 
 Rewards for delegators are also calculated in the exact same way that rewards for node operators are calculated, 
-with one difference in that 8% of the calculated amount is given to the node operator being delegated to. 
+with one difference in that 8% of the calculated amount is given to the delegated.
```
Comment (psiemens) [inline docs/content/staking/index.mdx:76]:
```suggestion
with one difference in that 8% of the calculated amount is given to the delegatee.
```

I believe _delegatee_ is the correct term here

File: docs/content/staking/technical-overview.mdx [hunk 4]
```diff
@@ -7,12 +7,12 @@ description: How to use FLOW for staking and delegation
 <Callout type="warning">
 NOTE: The details of the transactions described in this document are for regular interaction with the staking smart contracts. 
 If you or your users have locked tokens from the FLOW token sale and want to stake them, 
-please see the locked tokens section of the documentation to learn the process for staking or delegation.
+please see the Locked FLOW sections of the documentation to learn the process for staking or delegation.
```
Comment (psiemens) [inline docs/content/staking/technical-overview.mdx:10]:
```suggestion
please see the locked FLOW sections of the documentation to learn the process for staking or delegation.
```

**Review by psiemens — APPROVED** [review]

**Comment by joshuahannan** [comment]
Fixed comments in 7803169

**Pull request merged**